packages/taiko-client/cmd/flags/common.go

```diff
@@ -28,6 +28,12 @@ var (
 		Category: commonCategory,
 		EnvVars:  []string{"L1_WS"},
 	}
+	L1PrivateTxEndpoint = &cli.StringFlag{
+		Name:     "l1.privateTx",
+		Usage:    "Websocket RPC endpoint of a L1 private ethereum node",
+		Category: commonCategory,
+		EnvVars:  []string{"L1_PRIVATE_TX"},
```

Comment:
```suggestion
		EnvVars:  []string{"L1_PRIVATE_TX_WS"},
```

Reply:
Both http and ws are supposed to be supported.